modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -1890,21 +1905,6 @@
           end
         end
 
-        context 'when the activeDutyEndDate is Feb 29 in a leap year' do
-          let(:active_duty_end_date) { '2024-02-29' }
-
-          it 'responds with a 202' do
-            mock_ccg(scopes) do |auth_header|
-              json = JSON.parse(data)
-              json['data']['attributes']['serviceInformation']['servicePeriods'][0]['activeDutyEndDate'] =
-                active_duty_end_date
-              data = json.to_json
-              post submit_path, params: data, headers: auth_header
-              expect(response).to have_http_status(:accepted)
-            end
-          end
-        end
-
```

Comment:
Did not realize when I initially added this test it would fail once this date is no longer in the future, so removing to avoid another failing test on master once this date passes.